"use server"

import { handleError } from "../utils"

export const createCategory = async () => {
  try {;

    return;
  } catch (error) {
    handleError(error)
  }
}

export const getAllCategories = async () => {
  try {

    return;
  } catch (error) {
    handleError(error)
  }
}